index.html

```diff
@@ -0,0 +1,96 @@
+			<meta http-equiv="X-UA-Compatible" content="IE=edge">
+			<meta name="viewport" content="width=device-width, initial-scale=1.0">
+			<link rel="stylesheet" type="text/css" href="style-original.css">
+			<title>My potfolio</title>
+			<style>
+				@import url('http://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800;900&display=swap');
+			</style>
+			<link rel="stylesheet" href="style.css" type="text/css">
+			<script  type="text/javascript" scr="script.js"></script>
+			<link rel="preconnect" href="https://fonts.googleapis.com">
+			<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+			<link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+			<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+
+			<link href="svg.link" type="text/css">
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.css" integrity="sha512-FA9cIbtlP61W0PRtX36P6CGRy0vZs0C2Uw26Q1cMmj3xwhftftymr0sj8/YeezDnRwL9wtWw8ZwtCiTDXlXGjQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/brands.css" integrity="sha512-g9hNU97Dxb6qJZumYbKXC10tDP8J2rQdRxQoqona1LU0k0L8Hki+mOMHLbljf2TxSjgGl8f/PzUyGN3/XvA14w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+			<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/>
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/solid.css" integrity="sha384-Tv5i09RULyHKMwX0E8wJUqSOaXlyu3SQxORObAI08iUwIalMmN5L6AvlPX2LMoSE" crossorigin="anonymous"/>
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/fontawesome.css" integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous"/>
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/regular.min.css" integrity="sha512-k2UAKyvfA7Xd/6FrOv5SG4Qr9h4p2oaeshXF99WO3zIpCsgTJ3YZELDK0gHdlJE5ls+Mbd5HL50b458z3meB/Q==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+		</head>
+		<body>
+			<main>
+				<header class="header-background-mobile">
+          <nav id="AppBar-Compact-Dark-iOS">
+            <time id="Time-Style">
+              <time class="Time">9:45</time>
+            </time>
+            <div class="wifi">
+              <i class="fa-solid fa-wifi fa-xs"></i>
+            </div>	
+            <div class="sig-wf-bat">
+              <div class="signals">
+                <i class="fa-duotone fa-signal-bars-weak"></i>
```

Comment:
- [ ] As requested, kindly remove this section as it is not necessary. This would make your p[portfolio more professional.

![image](https://user-images.githubusercontent.com/98453979/205285331-57adad36-d1df-48f4-86e7-87f299a5aa29.png)

Reply:
@frank1738 Thanks, I now understand what the previous reviewer meant.